app/code/Magento/Swatches/view/base/web/js/swatch-renderer-tooltip.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'jquery',
+    'jquery-ui-modules/widget'
+], function ($) {
+    'use strict';
+
+    /**
+     * Render tooltips by attributes (only to up).
```

Comment:
Can you clarify what `(only to up)` means in this context?

Reply:
Oh nevermind, just saw that was copypasta from `swatch-renderer.js`. Is this whole file just copy + paste or are there further customizations?

Reply:
No, this is a simple copy paste.